src/sqlparser/tests/testdata/select.yaml

```diff
@@ -11,7 +11,7 @@
   formatted_sql: SELECT foo.v1.v2 FROM foo
 
 - input: SELECT ((((foo).v1)).v2) FROM foo
-  formatted_sql: SELECT (foo.v1.v2) FROM foo
+  formatted_sql: SELECT foo.v1.v2 FROM foo
```

Comment:
I don't quite understand meaning of this query, is this accessing a struct column v1, then accessing fields of the struct i.e. v2?

Is nesting required for it to be semantically / syntactically correct?

Reply:
Seems like this is for structs indeed. See `risingwave/src/frontend/test_runner/tests/testdata/struct_query.yaml`.